Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -1856,18 +1860,11 @@ void HTMLMediaElement::updateActiveTextTrackCues(const MediaTime& movieTime)
         // correctly identifies the type of the event, if the startTime is
         // less than the endTime in the cue.
         if (eventTask.second->startTime() >= eventTask.second->endTime()) {
-            auto enterEvent = Event::create(eventNames().enterEvent, Event::CanBubble::No, Event::IsCancelable::No);
-            scheduleEventOn(*eventTask.second, WTFMove(enterEvent));
-
-            auto exitEvent = Event::create(eventNames().exitEvent, Event::CanBubble::No, Event::IsCancelable::No);
-            scheduleEventOn(*eventTask.second, WTFMove(exitEvent));
+            executeCueEnterOrLeaveAction(*eventTask.second, CueAction::Enter);
```

Comment:
nit: It might be cleaner to make `eventTask` a structured binding `auto& [eventTime, eventCue]`

Reply:
Good point, we should do that as a followup.